i18n/src/main/resources/openfire_i18n.properties

```diff
@@ -2232,13 +2232,13 @@ setup.host.settings.domain=XMPP Domain Name:
 setup.host.settings.domain.help=The name of the XMPP domain that this server will be part of (for example: example.org). If this server will be part of an Openfire cluster, each server must have the same XMPP domain name.
 setup.host.settings.invalid_domain=Invalid domain.
 setup.host.settings.fqdn=Server Host Name (FQDN):
-setup.host.settings.fqdn.help=The fully qualified domain name of this server (for example: openfire.example.org). If this server will be part of an Openfire cluster, each server must have an unique server host name.
+setup.host.settings.fqdn.help=The fully qualified domain name of this server (for example: openfire.example.org). If this server will be part of an Openfire cluster, each server must have a unique server host name.
 setup.host.settings.invalid_fqdn=Invalid host name.
 setup.host.settings.port=Admin Console Port:
 setup.host.settings.secure_port=Secure Admin Console Port:
 setup.host.settings.invalid_port=Invalid port number
 setup.host.settings.port_number=Port number for the web-based admin console (default is 9090).
-setup.host.settings.secure_port_number=Port number for the web-based admin console through SSL (default is 9091).
+setup.host.settings.secure_port_number=Port number for the web-based admin console through TSL (default is 9091).
```

Comment:
```suggestion
setup.host.settings.secure_port_number=Port number for the web-based admin console through TLS (default is 9091).
```

Reply:
Oh, sorry, such a stupid mistake. Fixed to TLS